cohorts/views/views.py

```diff
@@ -179,7 +187,8 @@ def cohort_detail(request, cohort_id):
             'base_url': settings.BASE_URL,
             'cohort': cohort,
             'shared_with_users': shared_with_users,
-            'cohort_filters': cohort_filters
+            'cohort_filters': cohort_filters,
+            'cohort_version': str(cohort_versions[0])
```

Comment:
Per above you would change this to just

`'cohort_version': "; ".join(cohort_versions.get_displays())`

Reply:
Same as above. Fixed and confirmed works.